### PR TITLE
View CHANGELOG for version 2.2.13

### PR DESCRIPTION
**Pull request by necrox87**
Added "Exclude IPs" for exclude selected IPs from log activity

### CODE REVIEW

File: classes/class-aal-settings.php [hunk 1]
```diff
@@ -403,31 +419,31 @@ public static function select_field( $args ) {
...
 			return;
-		
+
 		?>
 		<select id="<?php echo esc_attr( $id ); ?>" name="<?php printf( '%s[%s]', esc_attr( $page ), esc_attr( $id ) ); ?>">
 			<?php foreach ( $options as $name => $label ) : ?>
-			<option value="<?php echo esc_attr( $name ); ?>" <?php selected( $name, (string) $value ); ?>>
-				<?php echo esc_html( $label ); ?>
-			</option>
+				<option value="<?php echo esc_attr( $name ); ?>" <?php selected( $name, (string) $value ); ?>>
```
Comment (KingYes) [inline classes/class-aal-settings.php]: Just remove the space from you open the PHP tag. the `selected()` function add later..

Comment (necrox87) [inline classes/class-aal-settings.php]: PHPStorm mistake... sorry

**Comment by necrox87** [comment]
As soon as possible I also want to implement a filter for subnets in addition to IPs and implement the "bootstrap input tags" plugin for the input entry.
bootstrap_input_tags

**Comment by necrox87** [comment]
Hi @KingYes you can resolve this merge conflicts?

**Comment by necrox87** [comment]
Fixed file conflicts

**Comment by arielk** [comment]
Sorry for the late reply (don't know how we missed it), but because of security issues we prefer not to merge this pr.

Thanks!

**Pull request closed without merging**